src/tests/ftest/util/soak_utils.py

```diff
@@ -142,21 +143,20 @@ def get_remote_dir(self, source_dir, dest_dir, host_list, shared_dir=None,
         shared_dir = self.sharedsoaktest_dir
     if append:
         for host in host_list:
-            shared_dir_tmp = shared_dir + append + "{}".format(host)
-            dest_dir_tmp = dest_dir + append + "{}".format(host)
+            shared_dir_tmp = shared_dir + append + f"{host}"
```

Comment:
`f"{host}"` is the same as `str(host)`. I think `str(host)` is better because it's straightforward